Classes/Backend/DynamicFlexForm.php

```diff
@@ -323,7 +320,10 @@ protected function recursivelyEvaluateClosures(array $dataStructureArray, array
      */
     protected function getCache()
     {
-        return GeneralUtility::makeInstance(CacheManager::class)->getCache('flux');
+        if ($this->cache !== null) {
```

Comment:
This happens once per field and at most a couple of times; cachemanager already caches the cache so we don't really win anything by this.